### ISSUE TITLE
Scroll to top doesn't include the adminbar's height

### ISSUE THREAD

**Issue by acosmin** [WordPress/twentytwenty#136]
and it will not scroll right to the very top

A quick fix would be to replace:
https://github.com/WordPress/twentytwenty/blob/c3fd9652e30fb489d83c493b4451ac361f81d75d/assets/js/construct.js#L496-L498

with:
```
var adminBar = $( '#wpadminbar' ),
    adminBarOffset = adminBar.length ? adminBar.innerHeight() : 0;

$( 'html, body' ).animate({
    scrollTop: scrollOffset - adminBarOffset
}, scrollSpeed );
```

**Comment by carolinan** [comment]
I believe this is awaiting the rewrite of the js file.

**Comment by hareesh-pillai** [comment]
This issue can be closed. This has been fixed in #163.
